components/layout/HeroBanner.tsx

```diff
@@ -1,175 +1,49 @@
-import { JSX } from '@emotion/react/jsx-runtime';
 import {
   Box,
-  Button,
+  Container,
   SxProps,
-  Theme,
   Typography,
-  TypographyProps,
 } from '@mui/material';
-import MuiMarkdown from 'mui-markdown';
 import React from 'react';
 
-import { AriaLabels } from '../../content/content.types.ts';
-import useLinkHandler from '../functional/LinkHandler.tsx';
-import ImageContainer from './ImageContainer.tsx';
-import SectionContainer from './SectionContainer.tsx';
+import EligibilityButton from '../helper/EligibilityButton.tsx';
 
 interface HeroBannerProps {
   header: string;
-  subheading?: string;
-  subheading2?: string;
-  imgsrc?: string;
-  ctaText?: string;
-  ctaLink?: string;
-  children?: JSX.Element | null;
   overrideStyles?: SxProps;
-  priority?: boolean;
-  ariaLabels?: AriaLabels;
 }
 
 const heroStyles: SxProps = {
-  backgroundColor: 'primary.dark',
+  backgroundImage: 'url(/illustrations/h1-home-desktop.png)',
```

Comment:
I think we will need to add the image to props and make it a var so because all pages use the hero banner and will need different images.

Reply:
Agreed. I refactored the component to use a background prop to do this